packages/mock-account-provider/app/entry.server.tsx

```diff
@@ -3,10 +3,10 @@ import type { EntryContext } from '@remix-run/node'
 import { Response } from '@remix-run/node'
 import { RemixServer } from '@remix-run/react'
 import { renderToPipeableStream } from 'react-dom/server'
-import { runSeed } from '../src/run_seed'
+import { runSeed } from './lib/run_seed'
 
 declare global {
-  let __seeded: boolean | undefined
+  var __seeded: boolean | undefined
```

Comment:
I found that this caused the pull_request PR check to fail but maybe you'll have better luck

Reply:
I did not